Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/ExposureStateUpdateWorker.kt

```diff
@@ -19,18 +18,18 @@ import timber.log.Timber
 class ExposureStateUpdateWorker @AssistedInject constructor(
     @Assisted val context: Context,
     @Assisted workerParams: WorkerParameters,
-    private val exposureResultStore: ExposureResultStore,
+    private val riskLevelStorage: RiskLevelStorage,
     private val enfClient: ENFClient,
     private val taskController: TaskController
 ) : CoroutineWorker(context, workerParams) {
 
     override suspend fun doWork(): Result {
         try {
             Timber.v("worker to persist exposure summary started")
-            enfClient.exposureWindows().let {
-                exposureResultStore.entities.value = ExposureResult(it, null)
-                Timber.v("exposure summary state updated: $it")
-            }
+//            enfClient.exposureWindows().let {
```

Comment:
Why is this commented out and not removed?

Reply:
Oversight. Fixed in next commit :+1: